File: src/Shared/Polyrific.Catapult.Shared.Dto/User/UserDto.cs
// Copyright (c) Polyrific, Inc 2018. All rights reserved.

namespace Polyrific.Catapult.Shared.Dto.User
{
    public class UserDto
    {
        public string Id { get; set; }

        /// <summary>
        /// Username of the user
        /// </summary>
        public string UserName { get; set; }

        /// <summary>
        /// First Name of the user
        /// </summary>
        public string FirstName { get; set; }

        /// <summary>
        /// Last Name of the user
        /// </summary>
        public string LastName { get; set; }

        /// <summary>
        /// Email address of the user
        /// </summary>
        public string Email { get; set; }

        /// <summary>
        /// Is user active?
        /// </summary>
        public bool IsActive { get; set; }

        /// <summary>
        /// Role of the user
        /// </summary>
        public string Role { get; set; }
    }
}

File: src/Web/Polyrific.Catapult.Web/ClientApp/src/app/core/models/account/user-dto.ts
export interface UserDto {
  id: number;
  userName: string;
  firstName: string;
  lastName: string;
  email: string;
  isActive: boolean;
  emailConfirmed: boolean;
  role: string;
  avatarFileId: number;
  externalAccountIds: { [key: string]: string };
}
